common-content/en/module/complexity/big-o/index.md

```diff
@@ -163,8 +174,12 @@ You will explore this theory in your backlog, but you will find that you already
   [LABEL=Quadratic Time]
 - Everyone at a party shaking hands with everyone else. If you double the number of people (n), the number of handshakes increases much faster (roughly n \* n). This is like nesting a loop inside a loop.
   [LABEL=Exponential Time]
-- Trying every possible combination to unlock a password. Each extra character dramatically increases the possibilities. This is like naive recursion;  we'll talk about this more later.
+- Trying every possible combination to unlock a password. Each extra character dramatically increases the possibilities.
   {{< /label-items >}}
 
 > [!TIP]
 >  Big-O notation also describes space complexity (how memory use grows). Sometimes an algorithm's time complexity is different from its space complexity. We have focused on time here, but you'll meet space complexity analysis in the assigned reading. 
+
+Big-O notation is focused on the _trend_ of growth, not the exact growth.
+
+If we think about strings, one character may take up one byte or four. If we think about doubling the length of the string, we don't worry about what characters are actually in the string. We think about the _trend_ - the string will take _about_ twice as much space. If the string only contained four-byte characters, and we only added one-byte characters, the string is still growing linearly, even though it may not take exactly double the space.
```

Comment:
```suggestion
Think about strings: one character may take up one byte or four. If we double the length of the string, we don't check which characters are in the string. We just think about the **trend**. The string will take _about_ twice as much space. If the string only has four-byte characters, and we add one-byte characters, the string is _still_ growing linearly, even though it may not take _exactly_ double the space.
```